### PR TITLE
[VEN-2712] OneJumpOracle deployment script and files for wstETH and weETH on arbitrum sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves VEN-2712

wstETH oneJumpOracle:
https://sepolia.arbiscan.io/tx/0x97a31597d80beea3d91964f04ac42530427e297649f0608fd59906936ce0da00

weETH oneJumpOracle:
https://sepolia.arbiscan.io/tx/0x60b89865dde3601c912192bb6dfa8c0321eb691582887a1003e52f519d766443

### CODE REVIEW

File: deploy/13-deploy-wstETH-weETH-oracles.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  await deploy("wstETHOneJumpChainlinkOracle", {
+    contract: "OneJumpOracle",
+    from: deployer,
+    log: true,
+    deterministicDeployment: false,
+    args: [wstETH, WETH, resilientOracle.address, chainlinkOracle.address],
+    proxy: {
+      owner: proxyOwnerAddress,
+      proxyContract: "OptimizedTransparentProxy",
```
Comment (chechu) [inline deploy/13-deploy-wstETH-weETH-oracles.ts:24]: We are using `OptimizedTransparentUpgradeableProxy` for the rest of the deployments in the oracle repo

File: deploy/13-deploy-wstETH-weETH-oracles.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const resilientOracle = await hre.ethers.getContract("ResilientOracle");
+  const chainlinkOracle = await hre.ethers.getContract("ChainlinkOracle");
+
+  await deploy("wstETHOneJumpChainlinkOracle", {
+    contract: "OneJumpOracle",
+    from: deployer,
+    log: true,
+    deterministicDeployment: false,
+    args: [wstETH, WETH, resilientOracle.address, chainlinkOracle.address],
+    proxy: {
```
Comment (chechu) [inline deploy/13-deploy-wstETH-weETH-oracles.ts:22]: We are setting the property `viaAdminContract` in the rest of the oracle deployments. Shouldn't we set it here too? /cc @GitGuru7

File: deploy/13-deploy-wstETH-weETH-oracles.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      owner: proxyOwnerAddress,
+      proxyContract: "OptimizedTransparentProxy",
+    },
+    skipIfAlreadyDeployed: true,
+  });
+};
+
+func.skip = async () => hre.network.name !== "arbitrumone" && hre.network.name !== "arbitrumsepolia";
+func.tags = ["wstETH_weETH_OneJumpOracles"];
```
Comment (chechu) [inline deploy/13-deploy-wstETH-weETH-oracles.ts:44]: I would add "arbitrum" in the tag and in the filename

File: deploy/13-deploy-wstETH-weETH-oracles.ts (hunk 2 above)
Comment (GitGuru7) [reply deploy/13-deploy-wstETH-weETH-oracles.ts:22]: The viaAdminContract is required for deploying upgradeable contracts on the zkSync network, and this specific oracle is deployed exclusively on Arbitrum, see [here](https://github.com/VenusProtocol/oracle/pull/214/files/f698c0c0843d58db57b8f50baa3b1d049edf670c#diff-860976e7cae6103ad5f743d19283d4fbb602120fa1cab425c0358a86a9cc09f0R43),  Therefore, zksync related changes are unnecessary .

Comment (GitGuru7) [reply deploy/13-deploy-wstETH-weETH-oracles.ts:22]: Again it is a choice. For the sake of consistency, we might need to update everywhere. Or just update this when needed in a zksync context. @chechu wdyt?

Comment (chechu) [reply deploy/13-deploy-wstETH-weETH-oracles.ts:22]: I would use `OptimizedTransparentUpgradeableProxy` and the `viaAdminContract` option, to be consistent with the deployments to zkSync (maybe in the future we need to deploy these contracts to that network). I wouldn't change the previous deployment scripts using `OptimizedTransparentProxy` now, to limit the scope of the change

File: deploy/13-deploy-wstETH-weETH-oracles.ts (hunk 1 above)
Comment (Debugger022) [reply deploy/13-deploy-wstETH-weETH-oracles.ts]: Deployed oracles with OptimizedTransparentUpgradeableProxy, a9966bc35c905f1bc84009af4f8a728ba2e1d98e

File: deploy/13-deploy-wstETH-weETH-oracles.ts (hunk 2 above)
Comment (Debugger022) [reply deploy/13-deploy-wstETH-weETH-oracles.ts:31]: Deployed oracles with OptimizedTransparentUpgradeableProxy using viaAdminContract, https://github.com/VenusProtocol/oracle/commit/a9966bc35c905f1bc84009af4f8a728ba2e1d98e

File: helpers/deploymentConfig.ts
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:1]: SHouldn't we add the config of new tokens [here](https://github.com/VenusProtocol/oracle/blob/6b9a2a58c1a8e35fb5e07be2a3b9adda3e9767a1/helpers/deploymentConfig.ts#L861) as well?

Comment (Debugger022) [reply helpers/deploymentConfig.ts:1]: It's used in the current script but I think we should add the addresses to maintain the consistency.

Comment (Debugger022) [reply helpers/deploymentConfig.ts:1]: Added, 520b24a650383aa87a613443d808f74a29674c36

**Review by chechu — APPROVED** [review]

**Pull request merged**